cf-agent/verify_files_utils.c

```diff
@@ -3596,13 +3596,13 @@ bool VerifyOwner(EvalContext *ctx, const char *file, const Promise *pp, Attribut
 
     if (attr.perms.groups->next == NULL && attr.perms.groups->gid == CF_UNKNOWN_GROUP)  // Only one non.existent item
     {
-        cfPS(ctx, LOG_LEVEL_ERR, PROMISE_RESULT_FAIL, pp, attr, "Unable to make file belong to an unknown group");
+        cfPS(ctx, LOG_LEVEL_ERR, PROMISE_RESULT_FAIL, pp, attr, "Unable to make file belong to an unknown group '%s'", attr.perms.groups->gid );
         *result = PromiseResultUpdate(*result, PROMISE_RESULT_FAIL);
```

Comment:
I'm saying that this is *not* emitting the user which is not found. Try it, it should always give you the same non-sensical number.